lib/services/azuredocdb/index.js

```diff
@@ -30,8 +30,8 @@ Handlers.provision = function(log, params, next) {
   if (!cp.allValidatorsSucceed()) {
     var reply = Reply(HttpStatus.INTERNAL_SERVER_ERROR);
     reply.value.description = 'Input validators failed.';
-    var err = new Error('Validation failed.');
-    err.code = HttpStatus.INTERNAL_SERVER_ERROR;
+    var err = new Error('Parameter validation failed.  Did you supply a DocDb parameters file?');
```

Comment:
I actually did learn to type on an actual typewriter.  Wasn't aware of the debate over the years.  Thanks for the reference.  Pretty funny stuff.  I'm good with the logic to use one space.  We'll see how good I am at changing my ways.